sympy/vector/functions.py

```diff
@@ -474,3 +474,54 @@ def _path(from_object, to_object):
         from_path.append(other_path[i])
         i -= 1
     return index, from_path
+
+def orthogonalize(vlist, orthonormal=False):
+    """
```

Comment:
Oh yeah this only works in Python 3. You'll have to just the `**kwargs` way to make ti Py 2 compatible.